### ISSUE TITLE
Entry with map not saving on first save of an entry

### ISSUE THREAD

**Issue by jpeacock** [ethercreative/simplemap#271]
### Description
When creating a new entry with a map field, the map data is not being saved in a way you can query in a template. 

### Steps to reproduce

1. Create a new entry with a Maps field in it, give it an address to look up, choose the address. Save entry.
2. Entry shows in listing. Address column is populated with correct/chosen address. 
3. Query the section with approximate location. No results 
4. Go into created entry and click "Save" a 2nd time. 
5. Entry can now be queried and shows on a results listing. 

I created a screen video of how to reproduce the issue:
https://www.dropbox.com/s/mur9fnmcycdrctd/maps-save-problem.mp4?dl=0

### Additional info

- Craft version: 3.5.7
- Maps version: 3.8.4.1
- PHP version: 7.4
- Database driver & version: MySQL 8
- Other Plugins: All other plugins disabled

**Comment by fthues** [comment]
Same problem here. The record in the `maps` table doesn't get created upon creating a new entry, only after resaving the entry. Couldn't figure out why I wasn't getting any results for a location query.

**Comment by Tam** [comment]
Could you share your `web.log` files?

**Comment by jpeacock** [comment]
> Could you share your `web.log` files?

Yep, here's a zip of my log file from saving an entry the first time in dev mode.
[web.log.zip](https://github.com/ethercreative/simplemap/files/5404980/web.log.zip)
